### PR TITLE
Services sort by

### PR DESCRIPTION
**Pull request by sgrg1**
Tim and I added filter , checked box.

### CODE REVIEW

File: src/components/ServiceList.jsx [hunk 1]
```diff
@@ -1,22 +1,61 @@
...
 import useServices from "../hooks/useServices";
+import CheckBox from "./CheckBox";
+import Search from "./Search";
 
 const ServiceList = () => {
   const { hasError, serviceItems = [], isLoading } = useServices();
+  const [isChecked, setChecked] = useState(false);
+  const [searchedItems, setSearchedItems] = useState([]);
+  const [searchText, setSearchText] = useState([]);
+  const [isFiltering, setIsFiltering] = useState(0);
```
Comment (martypowell) [inline src/components/ServiceList.jsx]: if you are using this naming convention, isFiltering should be a boolean value.

File: src/components/ServiceList.jsx [hunk 2]
```diff
@@ -1,22 +1,61 @@
...
-import React from "react";
+import React, { useState } from "react";
 import ServiceIconLink from "../components/ServiceIconLink";
 import useServices from "../hooks/useServices";
+import CheckBox from "./CheckBox";
+import Search from "./Search";
 
 const ServiceList = () => {
   const { hasError, serviceItems = [], isLoading } = useServices();
+  const [isChecked, setChecked] = useState(false);
```
Comment (martypowell) [inline src/components/ServiceList.jsx]: I'm assuming this is if popular services is checked?

Comment (sgrg1) [reply src/components/ServiceList.jsx]: yes, you are right

File: src/components/ServiceList.jsx (hunk 1 above)
Comment (sgrg1) [reply src/components/ServiceList.jsx]: done

**Pull request closed without merging**